File: AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/nativeads/carouselui/AppLovinActivityCallbacks.java
package com.applovin.enterprise.apps.demoapp.ads.applovin.nativeads.carouselui;

import android.app.Activity;

/**
 * Defines activity lifecycle methods which should be forwarded to an AppLovin view.
 */
public interface AppLovinActivityCallbacks
{
    /**
     * Override your activity or fragment's onResume method and invoke this method with a reference to the current activity.
     * @param activity Current activity.
     */
    public void onResume(final Activity activity);

    /**
     * Override your activity or fragment's onResume method and invoke this method with a reference to the current activity.
     * @param activity Current activity.
     */
    public void onStop(final Activity activity);
}

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/nativeads/carouselui/AppLovinActivityCallbacks.kt
package com.applovin.enterprise.apps.demoapp.ads.applovin.nativeads.carouselui

import android.app.Activity

/**
 * Defines activity lifecycle methods which should be forwarded to an AppLovin view.
 */
interface AppLovinActivityCallbacks
{
    /**
     * Override your activity or fragment's onResume method and invoke this method with a reference to the current activity.
     * @param activity Current activity.
     */
    fun onResume(activity: Activity)

    /**
     * Override your activity or fragment's onResume method and invoke this method with a reference to the current activity.
     * @param activity Current activity.
     */
    fun onStop(activity: Activity)
}
